### ISSUE TITLE
NoMethodError: undefined method `asset' for #<Array:0x00007f78c7582900>

### ISSUE THREAD

**Issue by wmakley** [rails/requestjs-rails#4]
This crash occurs when the engine loads, so on boot or running the install task.

Gemfile:
```ruby
gem 'rails', '~> 6.1.4', '>= 6.1.4.1'
# ...
# the following are updated to the latest versions as of this writing:
gem 'importmap-rails'
gem 'hotwire-rails'
gem 'requestjs-rails'
```

```
./bin/rails requestjs:install --trace
** Invoke requestjs:install (first_time)
** Execute requestjs:install
** Invoke requestjs:install:asset_pipeline (first_time)
** Execute requestjs:install:asset_pipeline
rails aborted!
NoMethodError: undefined method `asset' for #<Array:0x00007f78c7582900>
/Users/william/programming/coffee-log-rails/config/environment.rb:5:in `<main>'
/Users/william/programming/coffee-log-rails/bin/rails:5:in `<top (required)>'
/Users/william/programming/coffee-log-rails/bin/spring:10:in `require'
/Users/william/programming/coffee-log-rails/bin/spring:10:in `block in <top (required)>'
/Users/william/programming/coffee-log-rails/bin/spring:7:in `tap'
/Users/william/programming/coffee-log-rails/bin/spring:7:in `<top (required)>'
Tasks: TOP => app:template => environment
(See full trace by running task with --trace)
```

The actual crash appears to be in lib/requestjs/engine.rb, line 12::
```ruby

    initializer "requestjs.importmap" do
      if Rails.application.config.respond_to?(:importmap)
        Rails.application.config.importmap.paths.tap do |paths|
          paths.asset "@rails/request.js", path: "rails-requestjs" # line 12
        end
      end
    end
```

**Comment by evaneykelen** [comment]
I can confirm that using `gem 'requestjs-rails', github: 'rails/requestjs-rails', ref: 'refs/pull/3/head'` indeed solves the `NoMethodError`.

**Comment by marcelolx** [comment]
Should be fixed in https://github.com/rails/requestjs-rails/releases/tag/v0.0.6

Let me know if something is still broken!

**Comment by evaneykelen** [comment]
Solved with 0.0.6! Thanks!
